core/tx_pool.go

```diff
@@ -279,6 +284,8 @@ type TxPool struct {
 	initDoneCh               chan struct{}  // is closed once the pool is initialized (for tests)
 
 	changesSinceReorg int // A counter for how many drops we've performed in-between reorg.
+
+	isMiner atomic.Bool
```

Comment:
is atomic necessary?

Reply:
`eth.txPool = core.NewTxPool(config.TxPool, chainConfig, eth.blockchain)` is initialized before `ethBackend.TxPool().SetIsMiner(true)`, in case data racing, we can use atomic.